genassets.bash

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-go-bindata-assetfs -pkg "http" -nocompress -nomemcopy src/github.com/getlantern/ui/app/... && mv bindata_assetfs.go src/github.com/getlantern/flashlight/http
+tarfs -pkg http src/github.com/getlantern/ui/app > src/github.com/getlantern/flashlight/http/resources.go
```

Comment:
This is how one embeds resources with tarfs.  You have to `go install github.com/getlantern/tarfs/tarfs` for this to work.